lib/puppet/type/dism.rb

```diff
@@ -11,6 +11,20 @@
     desc 'The answer file for installing the feature.'
   end
 
+  newparam(:source) do
+    desc "The source files needed for installing the feature."
+  end
```

Comment:
No validation on source. Guess that's okay. It will either find the stuff or it will not.